py/desisim/scripts/lya_simqso_model.py

```diff
@@ -127,16 +127,20 @@ def EmLineTemplate_modified_develop(*args,**kwargs):
 
 def EmLineTemplate_modified(*args,**kwargs):
     kwargs.setdefault('scaleEWs',{
-                                  'Lyepsdel':1,
-                                  'CIII':0.3,
-                                  'NII':1.1,
-                                  'LyB/OIVn':0.7,
-                                  'LyB/OIVb':1.1,
-                                  'CIII*':1.8,
-                                  'LyAb':1.1,'LyAn':1.1
-                                  #Add more lines if needed.
-                                  })
-    kwargs['EmissionLineTrendFilename']=resource_filename('desisim', 'data/emlinetrends_Harris2016mod')
+                                'Lyepsdel':0.08,
+                                'CIII977':1.7,
+                                'NIII991':1.8,#2.2,
+                                #'unknownI':1.2,
+                                #'LyB/OIVn':1.2,'LyB/OIVb':1.2,
+                                'NII':0.8,
+                                'FeIII:UV1n':0.1,
+                                'FeIII:UV1b':0.1,
+                                'LyAb':1.2,'LyAn':1.2,
+                                'CIII*':1.7
+                                })
+    #kwargs['EmissionLineTrendFilename']=resource_filename('desisim', 'data/emlinetrends_Harris2016mod')
+    kwargs['EmissionLineTrendFilename']=resource_filename('desisim', 'data/emlinetrends_EDRv2')
```

Comment:
Maybe  we should call the file emlinetrends_Harris2016mod_v2, as this new file is a modified version of the previous one.